stock_storage_type/models/stock_location.py

```diff
@@ -497,19 +498,23 @@ def _get_sorted_leaf_child_locations(self, products):
         if not self.leaf_child_location_ids:
             return self.leaf_child_location_ids
         query = self._where_calc([("id", "in", self.leaf_child_location_ids.ids)])
-        _, where_clause, where_params = query.get_sql()
+        where_clause = query.where_clause
         orderby_clause, orderby_params = self._get_sorted_leaf_locations_orderby(
             products
         )
+        orderby = (
+            sql.SQL(orderby_clause, orderby_params)
```

Comment:
It should be `*orderby_params`, this is making `stock_storage_type_putaway_abc` failing.